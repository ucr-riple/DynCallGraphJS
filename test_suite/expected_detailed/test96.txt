{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:1:1:16:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:1:2:16:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:15:3:15:16)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:3:10:5:6)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:4:7:4:10)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test96.js:6:10:8:6)'
    ]
}