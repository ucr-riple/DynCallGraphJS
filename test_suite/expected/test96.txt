{ '(test97.js:1:1:16:5)':
   [ '(test97.js:1:2:16:2)' ],
  '(test97.js:15:3:15:16)':
   [ '(test97.js:3:10:5:6)' ],
  '(test97.js:4:7:4:10)':
   [ '(test97.js:6:10:8:6)' ] }